curriculum/challenges/english/25-front-end-development/review-html-accessibility/671a87a39b88245a579c2271.md

```diff
@@ -9,7 +9,46 @@ dashedName: review-html-accessibility
 
 Review the concepts below to prepare for the upcoming quiz.
 
+## Introduction to Accessibility
 
+- **Web Content Accessibility Guidelines**: The Web Content Accessibility Guidelines (WCAG) are a set of guidelines for making web content more accessible to people with disabilities. The four principles of WCAG are POUR which stands for Perceivable, Operable, Understandable, and Robust.
+
+## Assistive Technology for Accessibility
+
+- **Screen readers**: software programs that read the content of a computer screen out loud. They are used by people who are blind or visually impaired to access the web.
+- **Large text or braille keyboards**: used by people with visual impairments to access the web.
+- **Screen magnifiers**: software programs that enlarge the content of a computer screen. They are used by people with low vision to access the web.
+- **Alternative pointing devices**: used by people with mobility impairments to access the web. This includes devices such as joysticks, trackballs, and touchpads.
+- **Voice recognition**: used by people with mobility impairments to access the web. It allows users to control a computer using their voice.
+
+## Accessibility Auditing Tools
```

Comment:
Just a suggestion, I think you are leaving out by far the most popular automated accessibility testing tool, [axe DevTools](https://www.deque.com/axe/devtools/). In fact, that's what Lighthouse uses underneath the hood to do its testing. And it is also works in all browsers.

Reply:
I pulled this list of tools from the existing tools mention in the lectures. We could add the `axe DevTools` to the list but we will need to add a section for that in the corresponding lecture script.  It looks like the videos for this lecture block haven't been recorded yet, so I will need to alert the person recording this about this change

Reply:
Ya, no worries. I figured this list was dependent upon was was being taught. I just wanted to mention it because axe is by far the most popular tool for accessibility testing and nobody I know in accessibility uses Lighthouse.

@jdwilkin4, a bit off topic, I apologize, but is it possible to review the lecture scripts for this?

Reply:
Absolutely!

If you want to message me your email through private message on the forum or discord, I can send you the actual scripts copied to a Google doc so you can leave notes/or edit the copies directly. Then we can update the actual scripts.

The person recording the accessibility videos has been notified to hold off on recording these set of videos

I will give you list for the HTML and Accessibility lecture block as well as the CSS and Accessibility lecture block